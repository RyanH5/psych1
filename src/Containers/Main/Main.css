.main-container {
  padding-top: 100px;
  padding-bottom: 100px;
}

main {
  display: flex;
}