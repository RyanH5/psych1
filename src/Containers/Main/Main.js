import React from 'react';
import { NewsArticles } from '../NewsArticles/NewsArticles';
import DisplayPage from '../DisplayPage/DisplayPage';
import { Route } from 'react-router-dom';
import './Main.css';
import SignInPage from '../../Components/SignIn/SignIn';
import SignUpPage from '../../Components/SignUp/SignUp';

export const Main = () => {
  return (
    <div className="main-container">
      <Route
        exact path="/"
        component={DisplayPage}
      />
      <Route
        exact path="/signin"
        component={SignInPage}
      />
      <Route
        exact path="/signup"
        component={SignUpPage}
      />
      <Route
        exact path="/resilience"
        component={DisplayPage}
      />
      <Route
        exact path="/parenting"
        component={DisplayPage}
      />
      <Route
        exact path="/psychotherapy"
        component={DisplayPage}
      />
      <Route
        exact path="/speaking"
        component={DisplayPage}
      />
      <Route
        exact path="/family"
        component={DisplayPage}
      />
    </div>
  )
}