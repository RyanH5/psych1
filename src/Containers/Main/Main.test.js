import { Main } from './Main';
import { shallow } from 'enzyme';
import React from 'react';

describe('Main', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<Main />);
  
    expect(wrapper).toMatchSnapshot();
  })
})