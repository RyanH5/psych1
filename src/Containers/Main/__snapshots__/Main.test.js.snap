// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Route
          component={[Function]}
          exact={true}
          path="/"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/signin"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/signup"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/resilience"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/parenting"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/psychotherapy"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/speaking"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/family"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/contact"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/home"
/>,
        <Route
          component={[Function]}
          exact={true}
          path="/account"
/>,
      ],
      "className": "main-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/signin",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/signup",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/resilience",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/parenting",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/psychotherapy",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/speaking",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/family",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/contact",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/home",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "component": [Function],
          "exact": true,
          "path": "/account",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Route
            component={[Function]}
            exact={true}
            path="/"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/signin"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/signup"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/resilience"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/parenting"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/psychotherapy"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/speaking"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/family"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/contact"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/home"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/account"
/>,
        ],
        "className": "main-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/signin",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/signup",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/resilience",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/parenting",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/psychotherapy",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/speaking",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/family",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/contact",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/home",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/account",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
